app/workers/metrics/record_data_counts_worker.rb

```diff
@@ -9,6 +9,11 @@ def perform
         estimate = model.estimated_count
         Rails.logger.info("db_table_size", table_info: { table_name: model.table_name, table_size: estimate })
         DatadogStatsClient.gauge("postgres.db_table_size", estimate, tags: { table_name: model.table_name })
+
+        next unless model.const_defined?(:SEARCH_CLASS)
+
+        index_size = Search::Client.count(index: model::SEARCH_CLASS::INDEX_ALIAS).dig("count")
```

Comment:
Should we pull this into our `base.rb` search class and then use the model's SEARCH_CLASS to call it? Then we can customize it for FeedContent to allow us to filter those by model type in that index. 
```ruby 
model::SEARCH_CLASS.document_count

# In search/base.rb
def document_count 
  Search::Client.count(index: self.class::INDEX_ALIAS).dig("count")
end
```

Reply:
Updated. `FeedContent` isn't a model so it's not in the loop, but I'm sure we'll figure that out when the time comes.